src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -3081,72 +2750,6 @@ Resources:
 
             assertEqualLaunchConfigs(actual, expected)
 
-            assertFileText(
-                expected.templatePath,
-                `AWSTemplateFormatVersion: '2010-09-09'
-Transform: AWS::Serverless-2016-10-31
-Description: |
```

Comment:
Because of the structure of these tests, it's obvious what we gained here: now this configuration state doesn't exist anymore.

The tests are verbose because of all of the state that exists. The more state we remove (and thus improve the design), the nicer these tests become.